components/script/dom/vrstageparameters.rs

```diff
@@ -33,10 +33,10 @@ impl VRStageParameters {
             parameters: DOMRefCell::new(parameters),
             transform: Heap::default()
         };
+        // XXX unsound!
```

Comment:
Just to check, since it wasn't obvious to me. This is unsound because VRStageParameters is unrooted, so transform is not rooted too, right?